lib/convert.js

```diff
@@ -14,8 +15,32 @@ var converter = {
     currentFolder: {},
     env: {},
 
-    getMetaData: function(ramlString, callback) {
-        raml.load(ramlString).then(function(data) {
+    getMetaData: function(input, callback) {
+        var allFiles = input.allFiles,
+            reader = new ramlParser.FileReader(function (filePath) {
+                return new Promise(function (resolve, reject) {
+                    var targetFilePath = decodeURIComponent(filePath);

```

Comment:
Since for now we just need the title from the root raml string and other files are not required, we don't need use ramlParser. We can simple convert yaml to json and get the `title` property or we could just extract the title property from the ramlString.
@VShingala